### PR TITLE
Add note about supporting other OSes.

### PR DESCRIPTION
**Pull request by awholegunch**

### CODE REVIEW

File: cdap-docs/admin-manual/source/_includes/installation/installation.txt [hunk 1]
```diff
@@ -9,8 +9,11 @@ Preparing Package Managers
 CDAP components are available as either Yum ``.rpm`` or APT ``.deb`` packages. There is
 one package for each CDAP component, and each component may have multiple services.
 Additionally, there is a base CDAP package with three utility packages (for HBase
-compatibility) installed which creates the base configuration and the ``cdap`` user. We
-provide packages for *Ubuntu 12* and *CentOS 6*.
+compatibility) installed which creates the base configuration and the ``cdap`` user. 
+
+We provide packages for *Ubuntu 12* and *CentOS 6*. While these are the only packages we
+currently provide, they contain no OS-specific code, and the same packages will work on
```
Comment (wolf31o2) [inline cdap-docs/admin-manual/source/_includes/installation/installation.txt]: They do contain OS-specific code, even distribution-specific code. However, they do not contain distribution **version** specific code.

**Review by wolf31o2 — CHANGES_REQUESTED** [review]
Maybe it would be better to mention that these are the lower bound on supported versions.

We support CentOS 6.0+ or Ubuntu 12.04+

**Comment by dereklwood** [comment]
👍 lgtm

**Review by wolf31o2 — APPROVED** [review]
👍 LGTM

**Pull request merged**